RUN-15586 add tabs and new procedures (#734)
run-ai/docs: docs/admin/runai-setup/authentication/sso.md

## Step 1: UI Configuration

1. Press the `Tools & Settings` then press `General`.

> 2. Open the `Security` pane and press `Identity provider`.

Reviewer: @jasonnovichRunAI maybe we should write "+ identity provider"? which is actually what is written in the button?

```suggestion
2. Open the `Security` pane and press `+Identity provider`.
```

Author: ```suggestion
2. Open the `Security` pane and press `+Identity provider`.
```

3. Select the SSO protocol. Choose `SAML 2` or `Open ID Connect`.

!!! Note
    Use your SAML response file to fill in the fields below.

=== "SAML 2"

    1. Choose `From computer` or `From URL`.
    
    	1. For `From computer`, press the `Metadata XML file` field, then select your file for upload. 
    	2. For `From URL`, in the `Metadata XML Url` field, enter the URL to the XML Metadata file.
    

>     2. Copy the `Redirect URL` and `Entity ID` for use with your identity provider if required.

Reviewer: ```suggestion
    2. Copy the `Redirect URL` and `Entity ID` and use them in your identity provider.
```

    3. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
    4. When complete, press `Save`.

>     5. In the `Logout uri` field, enter the desired URL logout page. If left empty, you will be redirected to the Run:ai portal.

Reviewer: @jasonnovichRunAI I think this should go separatly because it is not under the identity provider / SAML

>     6. In the `Session timeout` field, enter the amount of idle time before users are automatically logged out. (Default is 60 minutes)

Reviewer: @jasonnovichRunAI I think this should go separatly because it is not under the identity provider / SAML

Author: I am not so sure, it would look strange to have these 2 lines sort of "disconnected" from the procedure.

    After you have configured the SAML 2 settings, you can download the XML file, and view the identity provider settings. 
	

> 	Press `Download` to download the file.

Reviewer: @jasonnovichRunAI is it referring to the download button in the closed card?

Author: Yes when the card/tile/pane is rolled up (closed) you have a download and edit and delete buttons.

	Pres `Edit` to both download the file, and view the:

    * Identity provider URL.
    * Identity provider entity ID.
    * Certificate expiration date.

=== "Open ID Connect"

    1. In the `Discovery URL` field, enter the discovery URL .
    2. In the `Client ID` field, enter the client ID.
    3. In the `Client Secret` field, enter the client secret.
    4. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
    5.When complete, press `Save`.

>     5. In the `Logout uri` field, enter the desired URL logout page. If left empty, you will be redirected to the Run:ai portal.

Reviewer: @jasonnovichRunAI I think this should go separatly because it is not under the identity provider / OIDC

Author: See comment above - I think it would look strange disconnected from the procedure,

>     6. In the `Session timeout` field, enter the amount of idle time before users are automatically logged out. (Default is 60 minutes)

Reviewer: @jasonnovichRunAI I think this should go separatly because it is not under the identity provider / OIDC

!!! Important Note

>     Upon pressing `Save`, all existing users will be rendered non-functional, and the only valid user will be the *Administrator email* entered above. You can always revert by disabling *Login via SSO*.

Reviewer: ```suggestion
    Upon pressing `Save`, all existing users will be rendered non-functional. You can always revert by deleting the identity provider.
```
